process/sandbox.md

```diff
@@ -8,13 +8,11 @@ The CNCF Sandbox is the entry point for early stage projects and has four goals:
 * Nurture projects (e.g. via CNCF [Service Desk](https://github.com/cncf/servicedesk) requests)
 * Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to CNCF legal, code of conduct and IP Policy requirements
 
-This proposal is to move all Inception level projects to the Sandbox and provide clarity on what Sandbox projects stand for.
+This document describes the Sandbox and provide clarity on what Sandbox projects stand for.
 
 ## What is the CNCF Sandbox
 
-There’s been a desire within the CNCF TOC and community to provide further clarity around project maturity levels in CNCF.
-
-When we initially created the Inception project level, it was intended to provide an avenue for technically interesting early-stage projects that were beneficial to the cloud-native community.  We are transitioning Inception projects to the Sandbox. Sandbox projects should be early-stage projects that the CNCF TOC believes warrant experimentation. The Sandbox should provide a beneficial, neutral home for such projects, in order to foster collaborative development. We aspire to make the Sandbox the preferred path for early-stage projects to enter the CNCF. More mature projects can continue to jump directly to incubation, but as the cloud-native ecosystem grows, we expect to see proportionally more early-stage projects.
+Sandbox projects should be early-stage projects that the CNCF TOC believes warrant experimentation. The Sandbox should provide a beneficial, neutral home for such projects, in order to foster collaborative development. We aspire to make the Sandbox the preferred path for early-stage projects to enter the CNCF. More mature projects can continue to jump directly to Incubation, but as the cloud-native ecosystem grows, we expect to see proportionally more early-stage projects.
```

Comment:
Can folks approach CNCF TOC with just an idea and not necessarily with a working code base? (can we please cover this scenario here as well?)

Reply:
We don't have any criteria expecting a working codebase at Sandbox level though we do expect some framework on what the project is planning to do e.g. public roadmap, contribution guidelines (see https://github.com/cncf/toc/pull/499#pullrequestreview-463189029)